magento/magento2
# Rename "billing_regione" to "billing_region"

### Description (*)
On sales_order_create_customer_block.xml column for billing region is missnamed "billing_regione" instead of "billing_region"

### Related Pull Requests


### Fixed Issues (if relevant)
1. fixes magento/magento2#26679: Billing_region field name on customer grid

### Manual testing scenarios (*)
1. On admin order creation in customer selection step
2. Check that billing_region column is correctly fill 

### Questions or comments


### Contribution checklist (*)
